types.py

```diff
@@ -0,0 +1,10 @@
+from LSP.plugin.core.protocol import URI, Location, Range
+from LSP.plugin.core.typing import TypedDict, List
+
+ShowReference = TypedDict('ShowReference', {
+    'references': List[Location],
+    'uri': URI,
+    'range': Range
+})
+
+ShowReferencesAction = List[ShowReference]
```

Comment:
Think this would be more conventional:
```suggestion
ShowReferencesParams = List[ShowReference]
```